File: security.js
let checkLogin = login => {
    if(login.length > 3 && login.length < 21) {
        if(login.match('[a-zA-Z0-9]')) {
            return true;
        }
    }
    return false;
}

let checkPassword = password => {
    if(password.length > 7 && password.length < 65) {
        if(password.match('[a-zA-Z0-9.()\#\_\*]')) {
            return true;
        }
    }
    return false;
}

let checkEmail = email => {
    return email.match(/^[0-9a-z-\.]+\@[0-9a-z-]{2,}\.[a-z]{2,}$/i);
}

module.exports.checkLogin = checkLogin;
module.exports.checkPassword = checkPassword;
module.exports.checkEmail = checkEmail;

File: security.py
import re
from hashlib import sha256

LOGIN_TEMPLATE = "[a-zA-Z0-9_]"
FBD_SYMBOLS = '<>@\''
EMAIL_TEMPLATE = "[^@]+@[^@]+\.[^@]+"

class Security:
    def checkLogin(self, login):
        if len(login) > 3 and len(login) < 14:
            if re.match(LOGIN_TEMPLATE, login) is not None:
                return True
        return False
    
    def checkPassword(self, password):
        return True
    
    def checkEmail(self, email):
        if re.match(EMAIL_TEMPLATE, email) is not None:
            return True
        print('wrong email')
        return False
    
    def checkText(self, text):
        if text == '': return False
        text = list(text)
        for symbol in list(FBD_SYMBOLS):
            if symbol in text:
                return False
        return True
    
    def sha256(self, value):
        return sha256(value.encode()).hexdigest()
